pdesolvers/heat_equation_1d/heat_equation.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+from matplotlib import pyplot as plt
+
+from pdesolvers.heat_equation_1d.strategy.solver_strategy import SolverStrategy
+
+
+class HeatEquation:
```

Comment:
It seems that this class is a `HeatEquationSolver` rather than a `HeatEquation`. Is that right?